.github/workflows/scorecard.yml

```diff
@@ -51,18 +52,18 @@ jobs:
           # For private repositories:
           #   - `publish_results` will always be set to `false`, regardless
           #     of the value entered here.
-          publish_results: false
+          publish_results: true
```

Comment:
Well, it seems this value is pointless given the results are published anyway, so what's the difference between true and false here?

Reply:
true means the results from this scan are published, rather than a [weekly scan done by the openssf team]. This includes a few extra checks, access to a badge, and raising of any issues found in the 'security tab'

Initially they weren't doing this on any of our repos. today AFAIK it's liboqs only.